css/justin-howard.css

```diff
@@ -0,0 +1,20 @@
+h1{
+  color: red;
+}
+#line1 {
+  color: green;
+}
+.date {
+  font-style: italic;
+  font-weight: bold;
+}
+li > blockquote {
+  text-decoration: underline;
+}
+body blockquote {
+  background-color: gray;
+}
+* {
```

Comment:
Best practice is have our universal selector rule (i.e. `* {}`) placed at the top of our CSS file.